src/hotspot/share/prims/stackwalk.cpp

```diff
@@ -189,46 +189,25 @@ int StackWalk::fill_in_frames(jint mode, BaseFrameStream& stream,
     LogTarget(Debug, stackwalk) lt;
```

Comment:
Nit, leftover.

Reply:
Thanks for the review.  Will clean up before it's integrated